config/routes.rb

```diff
@@ -1,3 +1,17 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/merchants', to: 'merchants#index'
+  get '/merchants/new', to: 'merchants#new'
+  get '/merchants/:id', to: 'merchants#show'
+  post '/merchants', to: 'merchants#create'
+  get '/merchants/:id/edit', to: 'merchants#edit'
+  patch '/merchants/:id', to: 'merchants#update'
+  delete '/merchants/:id', to: 'merchants#destroy'
+  get '/items', to: 'items#index'
+  get '/merchants/:id/items', to: 'items#list'
```

Comment:
technically, this is route should go to the items index for that merchant. So something you may consider in the future, is that when you have views that are very similar. In this case, the items index and the merchants items index (in your case, 'list'), you can combine these into one action. I'd have this route to 'items#index' and then in the controller if :merchant_id comes in (you'd have to change :id to :merchant_id to distinguish between what type of id is coming in), then you can save different values in the placeholder that would be passed to the same view.